ui/appui-react/src/appui-react/statusfields/message-center/MessageCenterField.tsx

```diff
@@ -0,0 +1,157 @@
+import "./MessageCenterField.scss";
+
+/**
+ * Type for Status state to satisfy NotificationMarker type checking
+ */
+type Status = "primary" | "negative";
+
+/** Message Center Field React component.
+ * @public
+ */
+export function MessageCenterField() {
+  const [messages, setMessages] = React.useState(MessageManager.messages);
+  const [notify, setNotify] = React.useState(false);
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [NotificationMarkerStatus, setStatus] =
+    React.useState<Status>("primary");
+
+  const indicatorRef = React.createRef<HTMLButtonElement>();
+  const title = UiFramework.translate("messageCenter.messages");
+
+  const handleOpenChange = (isOpenState: boolean) => {
+    setNotify(false);
+    setIsOpen(isOpenState);
+  };
+
+  const isProblemStatus = (message: NotifyMessageDetailsType): boolean => {
+    // See priority values in DgnPlatform defined in NotificationManager
+    return (
+      message.priority === OutputMessagePriority.Error ||
+      message.priority === OutputMessagePriority.Fatal
+    );
+  };
+
+  const determineStatus = () =>
+    messages.some((message) => isProblemStatus(message))
+      ? setStatus("negative")
+      : setStatus("primary");
+
+  React.useEffect(() => {
+    MessageManager.registerAnimateOutToElement(indicatorRef.current);
+
+    return MessageManager.onMessagesUpdatedEvent.addListener(() => {
+      messages.length > 0 ? setNotify(true) : setNotify(false);
+      setMessages(MessageManager.messages);
+      determineStatus();
+    });
+  });
+
+  const getMessages = (tab: string) => {
+    return [...messages].reverse().map((message, index) => {
+      if (messages.length > 0) {
+        const iconClassName = MessageManager.getIconClassName(message);
+        const iconSpec = MessageManager.getIconSpecFromDetails(message);
+        const text: MessageType = message.briefMessage;
+        if ((tab === "error" && isProblemStatus(message)) || tab === "all") {
+          return (
+            <MessageCenterMessage
+              key={index.toString()}
+              message={text}
+              details={message.detailedMessage}
+              icon={<Icon iconSpec={iconSpec} className={iconClassName} />}
+            />
+          );
+        }
+        // these returns are a work around to avoid 'Not all paths return a value error"
+        return <></>;
+      } else {
+        return (
+          <span className="uifw-message-prompt" key={`${index.toString()}`}>
+            No Messages.
+          </span>
+        );
+      }
+    });
+  };
+
+  const tabs = (
+    <>
+      {["all", "error"].map((tabType) => {
+        return (
+          <Tabs.Panel value={tabType} key={tabType}>
+            <div className=".uifw-statusFields-messageCenter-messageCenterField">
+              {getMessages(tabType)}
+            </div>
+          </Tabs.Panel>
+        );
+      })}
+    </>
+  );
+
+  return (
+    <>
+      <Popover
```

Comment:
I think we need a default placement. Should expand to the top and be centered w/ field if possible.